### PR TITLE
Minor formatting changes.

### PR DESCRIPTION
**Pull request by olliejth**

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#pylint:disable=line-too-long, invalid-name, broad-exception-caught,possibly-used-before-assignment,no-member
+# pylint:disable=line-too-long, invalid-name, broad-exception-caught,possibly-used-before-assignment,no-member
```
Comment (CalebMathers) [inline dashboard/dashboard.py:1]: Good space

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -13,15 +13,18 @@
 from email_validator import validate_email as ve, EmailNotValidError
 import phonenumbers
 from phonenumbers import NumberParseException
-from load_dashboard_data import connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name, load_celestial_body_information
+
+from load_dashboard_data import (connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name,
```
Comment (CalebMathers) [inline dashboard/dashboard.py:17]: Better way of organising imports on different lines

File: dashboard/dashboard.py [hunk 3]
```diff
@@ -147,11 +151,16 @@ def validate_phone_number(phone_number:str, region='GB') -> bool:
...
+            filtered_df['cloud_coverage_rolling'] = filtered_df['cloud_coverage_percent'].rolling(
+                window=24).mean()
+            filtered_df['visibility_rolling'] = filtered_df['visibility_m'].rolling(
+                window=24).mean()
+            filtered_df['temperature_rolling'] = filtered_df['temperature_c'].rolling(
+                window=24).mean()
+            filtered_df['precipitation_prob_rolling'] = filtered_df['precipitation_probability_percent'].rolling(
+                window=24).mean()
+            filtered_df['precipitation_mm_rolling'] = filtered_df['precipitation_mm'].rolling(
+                window=24).mean()
```
Comment (CalebMathers) [inline dashboard/dashboard.py:163]: Good formatting

File: dashboard/test
Comment (CalebMathers) [inline dashboard/test:1]: so long

**Review by CalebMathers — APPROVED** [review]

File: dashboard/test
Comment (AlexOBarnes) [reply dashboard/test:1]: Thanks for removing this

File: dashboard/dashboard.py [hunk 4]
```diff
@@ -405,33 +408,34 @@ def get_current_datetime() -> datetime:
...
-
                     # Commiting the valid user information to the 'subscriber' table in the RDS,
                     # while also mapping that subscriber instance to their associated county ID.
                     conn.commit()
                 st.success('✅ Your subscription has been added successfully!')
 
             # If there's an error from the database itself, it will be due to a UNIQUE schema violation.
             except psycopg2.errors.UniqueViolation:
-                st.error('🛑 Username already exists. Please choose a different username.')
+                st.error(
```
Comment (AlexOBarnes) [inline dashboard/dashboard.py:437]: Looks fab

**Review by AlexOBarnes — APPROVED** [review]

**Pull request merged**